addons/binding/org.openhab.binding.hdpowerview/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,30 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Hunter Douglas PowerView Binding
+Bundle-SymbolicName: org.openhab.binding.hdpowerview;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .,
+ lib/jcifs-1.3.18.jar
```

Comment:
I took a look at how Netbios works (purposefully avoiding looking at the jcifs implementation) and it doesn't seem simple. These older windows network protocols tend to be riddled with weird edge cases and workarounds. Will the LGPL license preclude its inclusion in OH as well? I can always drop it and lose discovery of the hub, at an obvious cost to usability.